### ISSUE TITLE
Create OpenTracing API compatibility bridge

### ISSUE THREAD

**Issue by alvarolobato** [elastic/apm-agent-go#131]
Implement lightweight OpenTracing API compatibility bridge.

**Comment by axw** [comment]
Extremely out of date, just for history: https://github.com/elastic/apm-agent-go/pull/17.

**Comment by alvarolobato** [comment]
Blocked by https://github.com/elastic/apm-agent-go/issues/174

**Comment by axw** [comment]
Merged into the v2 branch.
